src/agent/mod.rs

```diff
@@ -53,10 +60,30 @@ impl LoginOptions {
         self
     }
 
+    /// Define the redirect URL
```

Comment:
I just realized that by hiding internal api in #27, we are also hiding some documentation, so I'm preliminarily bringing it back here :)